docs/README.md

```diff
@@ -0,0 +1,299 @@
+toucan-sdk / [Exports](modules.md)
+
+# Toucan SDK
+
+[![NPM Package](https://img.shields.io/npm/v/toucan-sdk.svg)](https://www.npmjs.com/package/toucan-sdk)
+
+The Toucan SDK allows developers to build using Toucan's infrastructure tools on [Celo](https://celo.org/) and [Polygon](https://polygon.technology/). It provides a set of functions that allow developers to interact with Toucan Protocol's smart contracts and subgraphs. This includes redeeming of pool tokens and retirements of carbon credits, as well as querying data on projects and retirements.
+
+A Typescript SDK for [Toucan Protocol](https://toucan.earth/). Wraps around [Ethers.js](https://docs.ethers.io/v5/). Works in the web browser and Node.js.
+
+For the full documentation, check our [docs](https://docs.toucan.earth/toucan/dev-resources/sdk).
+
+⚠️ This SDK is heavily under development. USE AT YOUR OWN RISK. ⚠️
+
+## Install
+
+```
+npm i toucan-sdk
+```
+
+or
+
+```
+yarn add toucan-sdk
+```
+
+# Quickstart
+
+Instantiate the ToucanClient and set a `signer` & `provider` to interact with our infrastructure.
+
+```typescript
+import { ToucanClient } from "toucan-sdk";
+
+const toucan = new ToucanClient("alfajores", provider, signer);
```

Comment:
I think it would be great to link how to instantiate a provider / signer for Ethers or if using something like Wagmi, to import the signer/provider

Reply:
added to the main readme